x/transfermiddleware/keeper/keeper.go

```diff
@@ -164,7 +164,16 @@ func (keeper Keeper) HasAllowRlyAddress(ctx sdk.Context, rlyAddress string) bool
 	store := ctx.KVStore(keeper.storeKey)
 	key := types.GetKeyByRlyAddress(rlyAddress)
 
-	return store.Has(key)
+	if store.Has(key) {
+		return true
+	}
+
+	prefixStore := prefix.NewStore(store, types.KeyRlyAddress)
+	iter := prefixStore.Iterator(nil, nil)
+	defer iter.Close()
```

Comment:
if we want to permissionless relayer, we should remove `ibc_ante` instead of this.

Reply:
why?

this solution allows to configure to be permissionless as nèeded without node upgrade.

so now it permissioned on mainnet, but once decided it just gov voting to allow anybody to relay. right?

Reply:
it's a consensus break change so it always requires chain upgrades.

Reply:
how cli parameter of local folder can break consensus?

Reply:
and?

so deploy it with next upgrade. ypu anyway going to break prefix nwxt 2 weeks.

Reply:
and removal of filter all together is also breaking as  you suggested. why it is concern in this case?

Reply:
or you mean that  that adding removal of addresses to filter is breaking change?